composer.json

```diff
@@ -67,6 +67,8 @@
         "doctrine/doctrine-bundle": ">=3",
         "sonata-project/core-bundle": "<3.20",
         "sonata-project/doctrine-extensions": "<1.8",
+        "sonata-project/doctrine-mongodb-admin-bundle": "<3.5",
+        "sonata-project/doctrine-orm-admin-bundle": "<3.24",
```

Comment:
do we need to add this conflict here?

Reply:
If you use SonataAdmin 3.78 with an old persistence bundle, the `by_reference` option won't be passed.
It will lead to a behavior different than the actual one in 3.77.
People will have this bug (https://github.com/sonata-project/SonataAdminBundle/issues/6166) again.

I can remove the conflict if you think that's not a good reason.

Reply:
https://github.com/sonata-project/SonataAdminBundle/pull/6438#discussion_r507509807